js/authentication.md

```diff
@@ -561,6 +561,115 @@ export default class App extends React.Component {
 }
 ```
 
+#### Federated with Auth0
+
+You can use `Auth0` as one of the provider of your Cognito Federated Identity Pool so that you can allow your Auth0 users to have access to your AWS resources. 
```

Comment:
You can use `Auth0` as one of the providers in your Cognito Federated Identity Pool. This will allow users authenticated via Auth0 have access to your AWS resources.

Reply:
Done.